### ISSUE TITLE
Continuation of the project

### ISSUE THREAD

**Issue by pmagyar** [osresearch/hcpy#45]
Hello,

first of all. @osresearch Thank you for your great work. You have laid a great basis for a project with a lot of potential!

As you can see, there are people who want to use your project and are even committed to the point of wanting to participate in it themselves. However, I've been following the project for a while and I think it's not progressing as quickly as others would like.
This is not a criticism! Just an assessment of the situation.

As I saw in other projects, a good way the let the community and the project grow is to create a github organisation, add some people as owners and move the project to this organisation. In this way you can give your project to the people and the responsibility lies with several people and not just you.

Are you interested to go this way?
I would be happy if you answered me.

I wish you a nice weekend! 

Kind regards, 

Patrick

**Comment by aav7fl** [comment]
Bump @osresearch 🙏

**Comment by Meatballs1** [comment]
Maybe just fork and do it @pmagyar ? :) Would love to see this in home assistant

**Comment by pmagyar** [comment]
ok yes, you are right. It's been more than a month.
I will arrange everything.

give me some time.

**Comment by pmagyar** [comment]
so, this is it.

https://github.com/hcpy2-0/hcpy

Please create future issues and PRs in this repository.
If you want to be included in the organization, please write in the issue:
https://github.com/hcpy2-0/hcpy/issues/2

FYI @Meatballs1  @aav7fl @ericblade @meidlinga

**Comment by Meatballs1** [comment]
It doesn't look like its being picked up as a fork? So cant do a pull request to it from existing repos?

**Comment by pmagyar** [comment]
Yes i have let detached the fork through the github support. It is a new repo.

**Comment by miklosbagi** [comment]
Lol, I was just checking what's up with my PR #25 I proposed roughly 2 years ago, and came across this issue.  
There isn't much to signal that this repo isn't dead tbh.
